docs/contents/basic_usage.md

```diff
@@ -116,6 +116,28 @@ cargo install --force dmenv
 ```
 
 
+## Upgrading a top-level dependency
+
+Let's say your project depends on `foolib`. Version `1.3` works fine, but
+`1.4` just came out and even if you don't need any new feature from `1.4`,
+you'd like to check wether your project is compatible.
```

Comment:
is the second part of the sentence relevant?

Reply:
Yes, because if you *do* need a feature from 1.4, you should say so in the `setup.py`